zigup.zig

```diff
@@ -80,8 +80,15 @@ fn getHomeDir() ![]const u8 {
 }
 
 fn allocInstallDirString(allocator: Allocator) ![]const u8 {
-    // TODO: maybe support ZIG_INSTALL_DIR environment variable?
     // TODO: maybe support a file on the filesystem to configure install dir?
+    if(std.os.getenv("ZIG_INSTALL_DIR")) |install_dir| {
```

Comment:
`std.os.getenv` not working on Windows, but `try std.process.getEnvVarOwned` is cross-platform.